app/theme/client/imports/forms/input.css

```diff
@@ -149,27 +168,27 @@ textarea.rc-input__element {
 
 .rc-input-file {
 	position: relative;
-
+	
 	width: 100%;
-
+	
 	&__label {
 		display: block;
 
 		cursor: pointer;
 	}
-
+	
 	&__title {
 		font-size: var(--input-font-size);
 	}
-
+	
 	&__wrapper {
 		display: flex;
 		flex-direction: row;
-
+		
 		width: 100%;
 		margin: 0.5rem 0;
 		padding: 0.3rem;
-
+		
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4969942)